herd/top_herd.ml

```diff
@@ -2,7 +2,7 @@
 (*                           the diy toolsuite                              *)
 (*                                                                          *)
 (* Jade Alglave, University College London, UK.                             *)
-(* Luc Maranget, INRIA Paris-Rocquencourt, France.                          *)
```

Comment:
You might want to revert this before merging.

Reply:
Thanks!